app/decorators/application_choice_export_decorator.rb

```diff
@@ -0,0 +1,47 @@
+class ApplicationChoiceExportDecorator < SimpleDelegator
```

Comment:
I wonder about introducing `app/decorators/` (especially in the neutral namespace) to the project for the purpose of application exports, this seems like a provider-centric feature. 
I also wonder if it would be tidier to move this to `app/presenters/provider_interface/application_choice_export_presenter` or similar and adhere to the current pattern of services and presenters (and in this case sub-presenters). I like the delegation to `ApplicationChoice` but this could apply to a presenter too. Very much an opinion point, happy to discuss further.

Reply:
Yeah that's true, the main reason for this is that the export was asking for things that were the same as in the API. It feels important to enforce that those things are exactly the same.